dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/CompositionHost.cs

```diff
@@ -220,42 +185,21 @@ internal struct DispatcherQueueOptions
             public DISPATCHERQUEUE_THREAD_APARTMENTTYPE apartmentType;
         };
 
-        // Win32 method signature we're duplicating.
-        //HRESULT CreateDispatcherQueueController(
-        //  DispatcherQueueOptions options,
-        //  ABI::Windows::System::IDispatcherQueueController** dispatcherQueueController
-        //);
-        [DllImport("coremessaging.dll", EntryPoint = "CreateDispatcherQueueController", CharSet = CharSet.Unicode)]
+        [DllImport("coremessaging.dll")]
         internal static extern IntPtr CreateDispatcherQueueController(DispatcherQueueOptions options,
                                                 [MarshalAs(UnmanagedType.IUnknown)]
                                                out object dispatcherQueueController);
-
-
-        [DllImport("user32.dll", EntryPoint = "CreateWindowEx", CharSet = CharSet.Unicode)]
-        internal static extern IntPtr CreateWindowEx(int dwExStyle,
-                                                      string lpszClassName,
-                                                      string lpszWindowName,
-                                                      int style,
-                                                      int x, int y,
-                                                      int width, int height,
-                                                      IntPtr hwndParent,
-                                                      IntPtr hMenu,
-                                                      IntPtr hInst,
-                                                      [MarshalAs(UnmanagedType.AsAny)] object pvParam);
-
-        [DllImport("user32.dll", EntryPoint = "DestroyWindow", CharSet = CharSet.Unicode)]
-        internal static extern bool DestroyWindow(IntPtr hwnd);
-
+        
         #endregion PInvoke declarations
```

Comment:
>endregion [](start = 9, length = 9)

regions are kind of uncool these days. Would it be ok to get rid of them?

Reply:
removed, coolness +1

---
In reply to: [276434520](https://github.com/Microsoft/Windows.UI.Composition-Win32-Samples/pull/1#discussion_r276434520) [](ancestors = 276434520)